### ISSUE TITLE
NPE (EMF-NiCad)

### ISSUE THREAD

**Issue by dmazinanian** [tsantalis/JDeodorant#50]
The same old issue, the expression gap has no return type. 

Clone # 618 (Group 618, Pair 2-3): org.eclipse.emf.ecore.impl.EModelElementImpl#getEAnnotation (row 1978) and org.eclipse.emf.converter.ModelConverter#getReferredEPackage (row 1979)

```
java.lang.NullPointerException
    at ca.concordia.jdeodorant.eclipse.commandline.cloneinfowriter.CloneInfoCSVWriter.writeCloneInfo(CloneInfoCSVWriter.java:156)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.testRefactoring(Application.java:765)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.start(Application.java:185)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
```

**Comment by dmazinanian** [comment]
The same in Clone # 662 (Group 662, Pair 33-62): org.eclipse.emf.common.notify.impl.SingletonAdapterImpl#unsetTarget (row 2244) and org.eclipse.emf.ecore.util.BasicExtendedMetaData#rename (row 2273)

**Comment by tsantalis** [comment]
Fixed in commit [ecffbf1ef67acad5e90ae80f1e85ce5fee808436](https://github.com/tsantalis/JDeodorant/commit/ecffbf1ef67acad5e90ae80f1e85ce5fee808436)
